go/ct/gen/state.go

```diff
@@ -257,6 +257,14 @@ func (g *StateGenerator) SetBlockNumberOffsetValue(variable Variable, value int6
 	g.blockContextGen.SetBlockNumberOffsetValue(variable, value)
 }
 
+func (g *StateGenerator) RestrictToHaveBlobHash(variable Variable) {
```

Comment:
Please fix the grammar: maybe "RestrictToPresentBlobHashIndex"

Reply:
I actually thought the original name was good, since it is saying "restrict variable to have a blob hash". 
What if we drop the "Restrict" and just call them "PresentBlobHashIndex" and "AbsentBlobHashIndex" ?

Reply:
The thing is: the variable will not be mapped in the assignment to a hash but to an index that has a hash assigned to it in the blob-hash list. That's why I think "RestritToHaveBlobHash" is not fitting.